Configurations/windows-makefile.tmpl

```diff
@@ -452,9 +452,9 @@ list-tests:
 	@$(ECHO) "Tests are not supported with your chosen Configure options"
 	@{- output_on() if !$disabled{tests}; "\@rem" -}
 
-install: install_sw install_ssldirs install_docs
+install: install_sw install_ssldirs install_docs {- $disabled{fips} ? "" : "install_fips" -}
```

Comment:
Hmmm... looks like I added this change only to the unix Makefile in #13684 and forgot to add it to the Window Makefile as well. (I didn't touch VMS at all at that time because I left it to @levitte). Thanks for fixing it @t8m 

https://github.com/openssl/openssl/pull/13684/files#diff-d604626e1e710908bc96baae6ea27f3fc05cce645fb053cb6059ddd631eba670R528-R530